README.md

```diff
@@ -40,6 +40,19 @@ image:
 Additionally for the best user experience and optimized `sti` operation we suggest images
 to have `/bin/sh` and `tar` commands available.
 
+Pruning the contents of the source tree is possible if the user supplies a
+`.s2iignore` file in the root directory of the source repository.  Supply line by line the regular
+expression that best captures a set of files and directories you want removed.  You can also cite
```

Comment:
Maybe I'm picky, but I'd generally stay away from pruning/remove wording, because we are just ignoring/excluding those file from the build process, the actual removal that's happening under the hood does not matter for end user, imho.